.devcontainer/bin/postAttachCommand.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+# Run codegen
+deno task -c src/deno.json codegen
+deno task -c cli/deno.json codegen
+
+# .bashrc doesn't work when creating a new container.
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+export PATH="$PATH:$HOME/.bun/bin"
```

Comment:
It would be great if [completions for Deno](https://docs.deno.com/runtime/reference/cli/completions/) also are set up.

Reply:
It will register completions for Deno, by `.bashrc`. Please see 9da705aedb52f623e5ed2326d2dadd9db3db1e2d commit 🙏🏻